app/helpers/metadata_helper.rb

```diff
@@ -234,6 +234,12 @@ def validate_metadata_csv_for_samples(
     error_aggregator.set_metadata("num_cols", metadata["headers"].length)
     warning_aggregator = ErrorAggregator.new
 
+    # Require a header for every column.
+    if metadata["headers"].include? ""
```

Comment:
Could the headers include a nil value, or are they always strings?

Reply:
I believe they're always converted into strings when the file is read.